File: src/main/java/net/spartanb312/cursa/utils/GeometryMasks.java
package net.spartanb312.cursa.utils;

public class GeometryMasks {

    public static final class Quad {
        public static final int DOWN = 0x01;
        public static final int UP = 0x02;
        public static final int NORTH = 0x04;
        public static final int SOUTH = 0x08;
        public static final int WEST = 0x10;
        public static final int EAST = 0x20;
        public static final int ALL = DOWN | UP | NORTH | SOUTH | WEST | EAST;
    }

}

File: src/main/java/club/deneb/client/utils/GeometryMasks.kt
package club.deneb.client.utils

object GeometryMasks {

    object Quad {
        const val DOWN = 0x01
        const val UP = 0x02
        const val NORTH = 0x04
        const val SOUTH = 0x08
        const val WEST = 0x10
        const val EAST = 0x20
        const val ALL = DOWN or UP or NORTH or SOUTH or WEST or EAST
    }

}